SwiftUIHomeworks/SwiftUIHomeworks/Screens/FiltersView.swift

```diff
@@ -0,0 +1,195 @@
+// FiltersView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import SwiftUI
+
+/// Screen with filters for product
+struct FiltersView: View {
+    // MARK: - Private Constants
+
+    private enum Constants {
+        static let backImageName = "chevron.left"
+        static let pricingTitle = "Pricing"
+        static let colorsTitle = "Colors"
+        static let filtersTitle = "Filters"
+        static let sizeTitleValue: CGFloat = 25
+        static let offsetYBackButtonValue: CGFloat = -20
+        static let offsetYPricingValue: CGFloat = -45
+        static let offsetXBackButtonValue: CGFloat = -100
+        static let offsetYTitleFiltersValue: CGFloat = -20
+        static let offsetYColorsListValue: CGFloat = -90
+        static let paddingColorsListValue: CGFloat = -100
+        static let spacingColorsValue: CGFloat = 5
+        static let widthLineColorCircleValue: CGFloat = 2
+        static let totalWidthColorListValue: CGFloat = 50
+        static let amountColorsListColumnsValue: CGFloat = 4
+        static let filterImageName = "square.stack.3d.forward.dottedline.fill"
+        static let offsetXFilterValue: CGFloat = 100
+        static let categoryTitle = "Category"
+        static let moreTitle = "More"
+        static let forwardImageName = "chevron.forward"
+        static let offsetYDivideCategoryValue: CGFloat = -50
+        static let spacingProductScrollValue: CGFloat = 20
+        static let paddingProductScrollValue: CGFloat = 20
+        static let offsetYProductScrollValue: CGFloat = -50
+        static let offsetYDivideTitleValue: CGFloat = -50
+        static let offsetXDivideTitleValue: CGFloat = 20
+    }
+
+    // MARK: - Public properties
+
+    @State var products: [Product]
```

Comment:
```suggestion
    @State var products: [Product]
```
@State во вью модель

Reply:
Я на это вью передаю данные с другого экрана, это свойство нужно дня конструктор структуры